shared/pubsub.js

```diff
@@ -417,6 +422,11 @@ const defaultMessageHandlers = {
     // No need to reply.
   },
 
+  [NOTIFICATION_TYPE.KV] ({ channelID, key, data }) {
+    console.log(`[pubsub] Received KV update from channel ${channelID} ${key}:`, data)
```

Comment:
This should probably be `console.debug`

Reply:
Maybe, but then `console.log` is used for these types of logs elsewhere (see `main.js:228` and `pubsub.js:421`)